### ISSUE TITLE
New URN for tlg2703.tlg001.opp-grclat1.xml

### ISSUE THREAD

**Issue by annettegessner** [OpenGreekAndLatin/First1KGreek#214]
@AlisonBabeu:
It seems, this edition needs a new URN, since it's different from tlg2703.tlg001.opp-grc1 (see [Perseus Catalog](http://catalog.perseus.org/catalog/urn:cts:greekLit:tlg2703.tlg001.opp-grc1)). 

Here the link to a scan of the edition this file contains:
https://archive.org/details/corpusscriptorum02niebuoft

Since it's Greek text and Latin translation, I suggest calling the Greek textparts
tlg2703.tlg001.opp-grc2
and the separated Latin translation
tlg2703.tlg001.opp-lat2 .
Is that ok?

**Comment by AlisonBabeu** [comment]
Those URNs look perfect since it would be the second Greek edition of Anna Comnena's Alexias for the Catalog as well as the first Latin translation.

**Comment by sonofmun** [comment]
If it is the first Latin translation, should it be opp-lat1 instead of opp-lat2?

**Comment by AlisonBabeu** [comment]
You are quite right Matt, it should be opp-lat1, I think this is why in general I don't put full CTS-URNs in MODS records that I create but instead use work IDs and rely on the automatic system since clearly I cannot count.

**Comment by annettegessner** [comment]
Thank you!
So I'll use
tlg2703.tlg001.opp-grc2
and 
tlg2703.tlg001.opp-lat1 .
